services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from db.models import models
+
+from db.models import MovieSession
+
+import datetime
+
+
+def create_movie_session(
+        show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)

```

Comment:
The `CinemaHall` model is used here but not imported. Add `from db.models import CinemaHall` to the imports.